content/docs/contributing/to-docs/style-guide/media.md

```diff
@@ -26,21 +26,21 @@ Image output:
 
 The `image-width` shortcode adds an image with alternate text and restricts the image's width. The `image-width` shortcode can ensure image sizes are consistent within a topic, and that large images don't scale overly large in wide browser windows. Use this method when adding `.svg` diagrams, very large images, and in topics with multiple images where consistent image widths are desirable.
 
-`image-width` takes three double-quoted parameters in order:
+`image-width` takes three double-quoted named parameters:
 
-1. image link
-1. width
-1. alternate text
+1. `src="/images/<image.png>"` - Image file path.
+1. `width="<200>"` - Scale the image by specifying a width in pixels.
```

Comment:
I think you want either a placeholder description: `width="<image width>"` or an actual example: `width="200"`, but not the mixture of an actual example inside of angle brackets. I'm fine with either, but slightly prefer the placeholder description, for consistency with the placeholders used for the other parameters.
```suggestion
1. `width="<image width>"` - Scale the image by specifying a width in pixels.
```